src/markup.c

```diff
@@ -60,24 +82,21 @@ char *markup_transform(char *str, enum markup_mode markup_mode)
                 return NULL;
         }
 
-        if (markup_mode == MARKUP_NO) {
+        switch (markup_mode) {
+        case MARKUP_NULL:
+                assert(false);
```

Comment:
I agree with changing it to `markup != NULL`, but I like having it as a `case`.  Being able to look at the switch and see "yes, all of the values of the enum are handled" makes the code easier to read.